test/integration/take_part_page_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+require "gds_api/test_helpers/publishing_api_v2"
+require "gds_api/test_helpers/panopticon"
+
+class TakePartPageTest < ActiveSupport::TestCase
+  #api calls happen in after commit so we need to disable transactions
+  self.use_transactional_fixtures = false
+
+  setup do
+    DatabaseCleaner.clean_with :truncation
+    stub_any_publishing_api_call
+    @take_part_page = build(:take_part_page)
+  end
+
+  teardown do
+    DatabaseCleaner.clean_with :truncation
```

Comment:
I still think it's unnecessary :smile: 

This doesn't occur for free, and whitehall tests are getting slower by the day.